infrastructure/charts/agent/templates/apisixroute.yaml

```diff
@@ -32,6 +32,11 @@ spec:
       config:
         block_rules: ["_system/metrics"]
         rejected_message: "access to metrics resource is not allowed from an external location"
+    - name: request-id
+      enable: true
+      config:
+        header_name: "X-Request-ID"
```

Comment:
Would it be better to use the lowercase?
`x-request-id`

Reply:
X-Request-ID is the offical standard for this header (https://http.dev/x-request-id) - best to stick to it

Reply:
Yep, HTTP header names (aka. fields) are `supposed` to be case-insensitive as per the [spec](https://www.rfc-editor.org/rfc/rfc9110.html#section-5.1).

Reply:
Yes I agree X-Request-ID is standard ,  we should raise warning / handle in the code if we find in different case

Reply:
X-Request-ID is the official but lowercase is the standard...

(╯°□°）╯︵ ┻━┻)

But in the code when I'm using it is case insensitive.
I think we are fine we don't need to do any warning.